assets/src/edit-story/components/panels/karma/borderRadius.karma.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { useStory } from '../../../app/story';
+
+describe('Border Radius Panel', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  const getSelection = async () => {
+    const storyContext = await fixture.renderHook(() => useStory());
+    return storyContext.state.selectedElements;
+  };
+
+  describe('CUJ: Creator can Manipulate Shape: Border Radius', () => {
+    it('should allow the user to add border radius for text element', async () => {
+      await fixture.events.click(fixture.editor.library.textAdd);
+      // Choose Fill as background for visibility.
+      await fixture.events.click(
+        fixture.editor.inspector.designPanel.textStyle.fill.button
+      );
+
+      const panel = fixture.editor.inspector.designPanel.borderRadius;
+      await fixture.events.click(panel.radius('Top left'), { clickCount: 3 });
+      await fixture.events.keyboard.type('30');
+      await fixture.events.keyboard.press('tab');
+
+      const [element] = await getSelection();
+      // Since the default state is locked, verify that all the values were set to 30.
+      expect(element.borderRadius.topLeft).toBe(30);
+      expect(element.borderRadius.topRight).toBe(30);
+      expect(element.borderRadius.bottomLeft).toBe(30);
+      expect(element.borderRadius.bottomRight).toBe(30);
```

Comment:
Could we do these with `objectContaining`? Jasmine has the [same name and signature](https://jasmine.github.io/api/edge/jasmine#.objectContaining ) as [jest](https://jestjs.io/docs/en/expect#expectobjectcontainingobject) for this helper:

```js
      expect(element.borderRadius).toEqual(
        jasmine.objectContaining({
          topLeft: 30,
          topRight: 30,
          bottomLeft: 30,
          bottomRight: 30,
        })
      );
```

It'll result in a nicer diff if the assertion ever fails.

(and the same once below)